lib/thor/parser/option.rb

```diff
@@ -110,6 +101,26 @@ def #{type}?
 
   protected
 
+    def sample_banner
+      sample_banner = if banner && !banner.to_s.empty?
+                        "#{switch_name}=#{banner}".dup
+                      else
+                        switch_name
+                      end
+      required? ? sample_banner : "[#{sample_banner}]"
+    end
+
+    def inverse_sample
+      return unless boolean? && name !~ /^(force|no-.*)$/
```

Comment:
There's a double negative here :) Can we switch to using `if` to make it a bit simpler?